examples/acquiring_data/fido_metadata_queries.py

```diff
@@ -0,0 +1,59 @@
+"""
+====================================
+Querying Metadata clients using Fido
+====================================
+
+This example shows how to search and retrieve metadata using `~sunpy.net.Fido` from
+search facilities like `~sunpy.net.hek.HEKClient`, `~sunpy.net.helio.HECClient`,
+and `~sunpy.net.jsoc.JSOCClient`. It also shows how to display desired columns from the result.
+"""
+import os
+
+from sunpy.net import Fido
+from sunpy.net import attrs as a
+
+###################################################################
+# We will query Helio for the 'rhessi_flare_list' table.
```

Comment:
I would maybe describe briefly what Helio is and link it? https://www.helio-vo.eu/

```suggestion
# We will query the Heliophysics Integrated Observatory (`HELIO <https://www.helio-vo.eu/>`_) for the 'rhessi_flare_list' table.
```